flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/Fetcher.java

```diff
@@ -61,15 +53,15 @@
 	 *     }
 	 * }
 	 * }</pre>
-	 * 
+	 *
+	 * @param <T> The type of elements produced by the fetcher and emitted to the source context.
 	 * @param sourceContext The source context to emit elements to.
 	 * @param valueDeserializer The deserializer to decode the raw values with.
-	 * @param lastOffsets The array into which to store the offsets for which elements are emitted (operator state)
-	 * 
-	 * @param <T> The type of elements produced by the fetcher and emitted to the source context.
+	 * @param lastOffsets The map into which to store the offsets for which elements are emitted (operator state)
+*
```

Comment:
Strange formatting of `*`. Did you check whether the generic param should be before normal params or why did you move?